integrationtest/org/daisy/dotify/formatter/test/resource-files/toc-entry-on-resumed/toc-entry-on-resumed-document-range-input.obfl

```diff
@@ -37,18 +37,20 @@
       </pre-content>
    </volume-template>
    <sequence master="main">
-      <block id="ch_1" translate="">⠿ </block>
+      <block id="ch_1"/>
+      <block translate="">⠿ </block>
```

Comment:
please add a remark somewhere in the Java code: TODO allow references to blocks with content

Why does this not work, by the way?

Reply:
If the id were on the other block, it would be considered a "contentIdentifier" because the `startsContentIdentifiers()` is called after empty blocks have been discarded and before the first non-empty block is added to the page. A better solution would be to call `startsContentIdentifiers()` after the first row has been added to the page, although that is also not completely accurate because an identifier can happen in the middle of a row too.

Reply:
A different approach - but I would have to work this out - might be to have an end page number for every identifier. Right now, method `getPageNumber(String refid)` in `CrossReferenceHandler.java` returns the _first_ page number the block `refid` refers to. If we would have something like `getLastPageNumber(String refid)` that returns the _last_ page number of the block, you would be able to detect that a block starts in the previous volume and ends in the current volume.

Reply:
I don't see how that would help.

Reply:
Added a FIXME in https://github.com/mtmse/dotify.formatter.impl/pull/12/commits/b2d3d7aab426fdf0e53661985f3b4c390920433c